src/Conversionfunctions.jl

```diff
@@ -0,0 +1,23 @@
+module Conversionfunctions
+
+export Taitt
+import Convertible: findpath, haspath
+include("constants.jl")
+
+function Taitt(tai1::Float64, tai2::Float64)
```

Comment:
I completely agree on this.  A key point of porting the functions to Julia is that if the code is written in a general way, the functions will work with custom Julia types, opening the door to unexpected features.

I think it would be cool if this package will eventually be compatible with [`Measurements.jl`](https://github.com/JuliaPhysics/Measurements.jl), so that we can perform the conversion and propagate uncertainties at the same time.